lib/catalog/martyrology.ts

```diff
@@ -729,11 +776,16 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       dateOfDeath: 305,
       dateOfDeathIsApproximative: true,
     },
+    // src:
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureuse-marie-catherine-de-saint-augustin-vierge
+    // - https://en.wikipedia.org/wiki/Catherine_of_St._Augustine
     catherine_of_saint_augustine_de_simon_de_longpre_virgin: {
```

Comment:
According [English Wikipedia](https://en.wikipedia.org/wiki/Catherine_of_St._Augustine) and [French Wikipedia](), her full:

- religious name is `Mary Catherine of Saint Augustine` (note: religious names seem to be localised, therefore in English `Marie` became `Mary` and the hyphen was removed in comparison with its French version);
- name is `Catherine de Simon de Longpré`;

therefore, the ID should be `mary_catherine_of_saint_augustine_catherine_de_simon_de_longpre_virgin`.

```suggestion
    mary_catherine_of_saint_augustine_catherine_de_simon_de_longpre_virgin: {
```